ReSource.asm

```diff
@@ -45723,6 +45732,8 @@ lbC0255CE	move.b	#$2C,(a4)+
 	addq.w	#4,d6
 	rts
 
+ext_mc68060	lea	(MC68060.MSG,pc),a0
```

Comment:
Do you want to reuse this sub?
Otherwise I don't see a reason for it.

Reply:
It could be used for plpar/w instructions.

Reply:
ok fine